cli/gen_runtime_vectors/ledger-paratime-transactions.md

```diff
@@ -0,0 +1,648 @@
+| Path[0]    | byte (4)       | Derivation Path Data   | 44             |
+| Path[1]    | byte (4)       | Derivation Path Data   | 0              |
+| Path[2]    | byte (4)       | Derivation Path Data   | ?              |
+| Path[3]    | byte (4)       | Derivation Path Data   | ?              |
+| Path[4]    | byte (4)       | Derivation Path Data   | ?              |
+
+First three items in the derivation path will be hardened automatically hardened
+
+#### Response
+
+| Field   | Type      | Content               | Note                     |
+| ------- | --------- | --------------------- | ------------------------ |
+| PK      | byte (32) | Public Key            |                          |
+| ADDR    | byte (??) | Hex addr              |                          |
+| SW1-SW2 | byte (2)  | Return code           | see list of return codes |
+
+### SIGN_PT_ED25519
+
+#### Command
+
+| Field | Type     | Content                | Expected  |
+| ----- | -------- | ---------------------- | --------- |
+| CLA   | byte (1) | Application Identifier | 0x05      |
+| INS   | byte (1) | Instruction ID         | 0x05      |
+| P1    | byte (1) | Payload desc           | 0 = init  |
+|       |          |                        | 1 = add   |
+|       |          |                        | 2 = last  |
+| P2    | byte (1) | ----                   | not used  |
+| L     | byte (1) | Bytes in payload       | (depends) |
+
+The first packet/chunk includes only the derivation path
+
+All other packets/chunks should contain message to sign
+
+*First Packet*
+
+| Field      | Type     | Content                | Expected  |
+| ---------- | -------- | ---------------------- | --------- |
+| Path[0]    | byte (4) | Derivation Path Data   | 44        |
+| Path[1]    | byte (4) | Derivation Path Data   | 474       |
+| Path[2]    | byte (4) | Derivation Path Data   | ?         |
+| Path[3]    | byte (4) | Derivation Path Data   | ?         |
+| Path[4]    | byte (4) | Derivation Path Data   | ?         |
+
+*Other Chunks/Packets*
+
+| Field   | Type     | Content              | Expected |
+| ------- | -------- | -------------------- | -------- |
+| Data    | bytes... | Context+Message+Meta |          |
+
+Data is defined as:
+
+| Field   | Type     | Content                    | Expected     |
+| ------- | -------- | -------------------------- | ------------ |
+| CtxLen  | byte     | Context Length             |              |
+| Context | bytes..  | Context                    | CtxLen bytes |
+| Message | bytes..  | CBOR data to sign          |              |
+| Meta    | bytes..  | CBOR metadata to verify    |              |
+
+#### Response
+
+| Field   | Type      | Content     | Note                     |
+| ------- | --------- | ----------- | ------------------------ |
+| SIG     | byte (64) | Signature   |                          |
+| SW1-SW2 | byte (2)  | Return code | see list of return codes |
+
+### SIGN_PT_SR25519
```

Comment:
I used the existing [APDU protocol reference](https://github.com/Zondax/ledger-oasis/blob/master/docs/APDUSPEC.md) from Zondax as a base. I would stick to this format since we can just copy&paste it from our ADR to the reference then.